docs/mcd.md

```diff
@@ -0,0 +1,326 @@
+# Modular Connection Dialog (MCD)
+
+**IMPORTANT:** This is alpha software and should be used in a test environment. Do not deploy the connector to a production environment.
+
+MCD is a new feature that enables a more fully data-driven connection dialog for plug in connectors. Additionally, it provides some control over the metadata hierarchy elements--Database, Schema, and Table--both in the connection dialog and in the schema viewer, which the user sees after the connection is established. 
+
+This feature is new in 2020.2. It is an alpha feature, and as such it is available in Desktop only. In future releases it will become available in other Tableau products including Server and Prep. 
+
+# How to Use MCD
+
+To use MCD, in the manifest replace `<connection-dialog>`, which references a .tcd file, with `<connection-fields>`, which references a .xml file, described below. The connector will fail to load if both elements appear in the manifest. 
+
+If you wish to modify metadata hierarchy behavior you can add to the manifest a `<connection-metadata>` element, which also references a .xml file and is described below. 
```

Comment:
What is the behavior if you don't have this file? Below, it seems like the default for database, schema, and table is to not show that hierarchy, so by that logic for a connector to work it would need this file.

Reply:
Added a sentence to be explicit about it.